configurations/__init__.py

```diff
@@ -32,3 +32,4 @@ def setup(app=None):
     in your docs' ``conf.py``.
     """
     _setup()
+    return {'version': __version__, 'parallel_read_safe': True}
```

Comment:
The setup function is used for other purposes as well, e.g. Celery support as in http://django-configurations.readthedocs.org/en/stable/cookbook/?highlight=setup#celery and relates to Django's equally named function `django.setup`, so it'd be odd to return Sphinx specific values here.

I suggest to add a new module `configurations.sphinx` with a `setup` function that returns this and calls `configurations._setup` as well. This needs a documentation update and a special note in the changelog I suppose.